Repair open metadata editor if images are available

Apparently, as soon as images are in the process, opening metadata editor no longer works; a white page appears. This stack trace has been preserved:

```
[ERROR] DataEditorForm - errorLoadingOne
org.kitodo.data.database.exceptions.DAOException: Object cannot be found in database
at org.kitodo.data.database.persistence.ProcessDAO.getById(ProcessDAO.java:25)
at org.kitodo.data.database.persistence.ProcessDAO.getById(ProcessDAO.java:19)
at org.kitodo.production.services.data.base.SearchDatabaseService.getById(SearchDatabaseService.java:140)
at org.kitodo.production.forms.dataeditor.DataEditorForm.getCurrentTaskOptions(DataEditorForm.java:503)
```

**Goal:** Metadata editor can be opened.

--- a/Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java
+++ b/Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java
@@ -22,6 +22,7 @@ import java.util.LinkedList;
 import java.util.List;
 import java.util.Locale.LanguageRange;
 import java.util.Map;
+import java.util.NoSuchElementException;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
@@ -204,7 +205,7 @@ public class DataEditorForm implements RulesetSetupInterface, Serializable {
             selectedMedia = new LinkedList<>();
             init();
             openProcesses.put(process.getId(), user);
-        } catch (IOException | DAOException | InvalidImagesException e) {
+        } catch (IOException | DAOException | InvalidImagesException | NoSuchElementException e) {
             Helper.setErrorMessage(e.getLocalizedMessage(), logger, e);
             return referringView;
         }

